go/libraries/doltcore/doltdb/system_variables.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package doltdb
```

Comment:
This shouldn't go in the doltdb package, should go in sqle

Reply:
that's what i did originally but it created import cycles. subpackage of sqle?

Reply:
what about in env?